ocs_ci/ocs/registry.py

```diff
@@ -0,0 +1,209 @@
+import logging
+import time
+import base64
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.utils import run_cmd
+from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+def change_registry_backend_to_ocs():
+    """
+    Function to deploy registry with OCS backend.
```

Comment:
Add section:
```
Raises:
    AssertionError: When ...

```

Reply:
ack, will update all the comments related to adding new line, Args, Returns and Raises